app/controllers/catalog_controller.rb

```diff
@@ -122,8 +122,7 @@ class CatalogController < ApplicationController
     add_common_date_facet_fields_to_config! config
 
     # This will help us find records that need to be fixed before we can move to cocina.
-    config.add_facet_field 'source_id_errors_ssim', label: 'Source ID Errors', home: false,
```

Comment:
I get the addition but why the removal of this facet?  It's not mentioned in the PR description or the ticket.

Reply:
It was renamed because it now has more than source id errors.

Reply:
ah!  thank you.